lib/OnyxUtils.ts

```diff
@@ -661,7 +662,8 @@ function keysChanged<TKey extends CollectionKeyBase>(
 
         // Regular Onyx.connect() subscriber found.
         if (typeof subscriber.callback === 'function') {
-            if (!notifyConnectSubscribers) {
+            // Check if it's a useOnyx or a regular Onyx.connect() subscriber
+            if ((subscriber.isUseOnyxSubscriber && !notifyUseOnyxHookSubscribers) || (!subscriber.isUseOnyxSubscriber && !notifyConnectSubscribers)) {
```

Comment:
```suggestion
            if (!notifyUseOnyxHookSubscribers || !notifyConnectSubscribers) {
```
Could be just that, no?

Reply:
Not really, in the initial condition we check:
- Part 1 `(subscriber.isUseOnyxSubscriber && !notifyUseOnyxHookSubscribers)` - if it's an onyx subscriber, and if it should be skipped
- Part 2 `(!subscriber.isUseOnyxSubscriber && !notifyConnectSubscribers)`  - if it's **not** an onyx subscriber, and if it should be skipped